forge-gui/res/cardsfolder/upcoming/uchuulon.txt

```diff
@@ -3,10 +3,12 @@ ManaCost:3 B
 Types:Creature Crab Ooze Horror
 PT:*/4
 S:Mode$ Continuous | EffectZone$ All | CharacteristicDefining$ True | SetPower$ X | Description$ CARDNAME's power is equal to the number of Crabs, Oozes, and/or Horrors you control.
-T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | IsPresent$ Card.Creature+OppOwn | PresentZone$ Graveyard | PresentCompare$ GE1 | Execute$ TrigExile | TriggerDescription$ Horrific Symbiosis — At the beginning of your end step, exile up to one target creature card from an opponent's graveyard. If you do, create a token that's a copy of Uchuulon.
-SVar:TrigExile:DB$ ChangeZone | Origin$ Graveyard | Destination$ Exile | ValidTgts$ Creature.OppOwn | TgtPrompt$ Select up to one target creature card from an opponent's graveyard | TargetMin$ 0 | TargetMax$ 1 | SubAbility$ DBCopy
-SVar:DBCopy:DB$ CopyPermanent | Defined$ Self | NumCopies$ 1
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigExile | TriggerDescription$ Horrific Symbiosis — At the beginning of your end step, exile up to one target creature card from an opponent's graveyard. If you do, create a token that's a copy of Uchuulon.
+SVar:TrigExile:DB$ ChangeZone | Origin$ Graveyard | Destination$ Exile | ValidTgts$ Creature.OppOwn | TgtPrompt$ Select up to one target creature card from an opponent's graveyard | TargetMin$ 0 | TargetMax$ 1 | RememberChanged$ True | SubAbility$ DBCopy
+SVar:DBCopy:DB$ CopyPermanent | Defined$ Self | NumCopies$ Y | SubAbility$ DBCleanup
```

Comment:
@tool4ever is cleanup needed?

Reply:
yea otherwise it might always copy after the first time